style.css

```diff
@@ -0,0 +1,390 @@
+@font-face {
+  font-family: NexaLight;
+  src: url('./Font/NexaLight.otf');
+}
+
+@font-face {
+  font-family: NexaBlack;
+  src: url('./Font/NexaBlack.otf');
+}
+
+@font-face {
+  font-family: NexaBold;
+  src: url('./Font/NexaBold.otf');
+}
+
+* {
+  margin-top: 10px;
+}
+
+.banner-section {
+  background-color: black;
+  display: flex;
+  margin: auto;
+  position: relative;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+}
+
+.container-banner {
+  display: flex;
+}
+
+.main-title {
+  display: none;
+  font-size: 25px;
+  line-height: 63px;
+  color: black;
+  margin: 0 0 15px;
+}
+
+.banner-wrapper {
+  display: flex;
+}
+
+.left-section {
+  width: 50%;
+  text-align: left;
+  margin: 50px 0px 40px 40px;
+  flex: 1;
+}
+
+.banner-image {
+  max-width: 400px;
+  width: 100%;
+  align-self: flex-start;
```

Comment:
why this here without flex @aeshapatel025

Reply:
changes done